start.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/sh
 
+set -eu
+
+if [ -n "${PULP_DEPLOYMENT:-}" ]
+then
+  ./manage check --deploy --fail-level WARNING
```

Comment:
Am I understanding right that this means local dev environments which don't have `PULP_DEPLOYMENT` set will fail if any `WARNING` is emitted?

Reply:
The idea here is that we fail deploying if some things are not configured insecure or look like local dev.
It should only run if the variable was not empty.